src/resolvers/Mutation/createChat.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ORGANIZATION_NOT_FOUND_ERROR,
+  USER_NOT_FOUND_ERROR,
+} from "../../constants";
+import { errors, requestContext } from "../../libraries";
+import { Chat, Organization, User } from "../../models";
+import { cacheOrganizations } from "../../services/OrganizationCache/cacheOrganizations";
+import { findOrganizationsInCache } from "../../services/OrganizationCache/findOrganizationsInCache";
+import type { MutationResolvers } from "../../types/generatedGraphQLTypes";
+/**
+ * This function enables to create a chat.
+ * @param _parent - parent of current request
+ * @param args - payload provided with the request
+ * @param context - context of entire application
+ * @remarks The following checks are done:
+ * 1. If the user exists
+ * 2. If the organization exists
+ * @returns Created chat
+ */
+export const createChat: MutationResolvers["createChat"] = async (
+  _parent,
+  args,
+  context,
+) => {
+  let organization;
+  const usersInChat = [];
+  if (args.data.isGroup && args.data.organizationId) {
+    const organizationFoundInCache = await findOrganizationsInCache([
+      args.data.organizationId,
+    ]);
+
+    organization = organizationFoundInCache[0];
+
+    if (organizationFoundInCache.includes(null)) {
```

Comment:
`organizationFoundInCache.includes(null)` is redundant because we’re only interested in the first organization `(organizationFoundInCache[0])`. It could lead to confusion if you were checking multiple organizations, but in this case, it's unnecessary. Directly check the first result from the cache.